### ISSUE TITLE
Mitigating Neural Network Overconfidence with Logit Normalization

### ISSUE THREAD

**Issue by tma15** [tma15/paper-reading-list#166]
https://arxiv.org/pdf/2205.09310.pdf
